app/controllers/dashboards_controller.rb

```diff
@@ -22,6 +22,10 @@ def search
   end
 
   def budget_bar
+    # i18n-tasks-use t('dashboard.budget_bar.pledged_item')
+    # i18n-tasks-use t('dashboard.budget_bar.sent_item')
+    # i18n-tasks-use t('dashboard.budget_bar.pledged_report')
+    # i18n-tasks-use t('dashboard.budget_bar.sent_report')
```

Comment:
Do we wanna clean these up, or are they place-holding for a soon-to-come thing?

Reply:
ah, thanks for calling this out -- we call on these via interpolation, so i18n health thinks they aren't being used. By declaring them in the controller we're sending a `noqa` type message to the health checker, letting it know that ACTUALLY we're using these, just indirectly. I'll add a comment.